### ISSUE TITLE
Allow test suite to wait for a length of time with no writes to the Node.

### ISSUE THREAD

**Issue by nemanja-boric-sociomantic** [sociomantic-tsunami/turtle#50]
Many applications are architectured in a way where some predefined input has been feed into them, process the input and output the profiles into, say, DHT. It's often necessary to know that the application has finished all processing and that all writes landed into the output node before confirming the contents of it.

One of the easiest ways, which may be just enough for most purposes is to allow turtle to wait for a length of time with no changes made to the whatever fake node.

**Comment by nemanja-boric-sociomantic** [comment]
Removed the assigned milestone to let @mihails-strasuns-sociomantic assign it, if this ticket is valid.

**Comment by mihails-strasuns-sociomantic** [comment]
One clarification - in this case `expectRecordChange` is not suitable because it is not known which  record will change last?

**Comment by nemanja-boric-sociomantic** [comment]
That's correct - the user may not know the last record. Am I right @jens-mueller-sociomantic ?

**Comment by jens-mueller-sociomantic** [comment]
Yes, you're right.
Note though, that it is currently unclear whether turtle is the way to go for my current use case. Maybe consult with others if they need/want this feature.

**Comment by mihails-strasuns-sociomantic** [comment]
That would be indeed curious to hear. In general that sounds a bit out of line with turtle purpose (very fine-grained application testing) as one is expected to know somewhat precisely which records will change in what order - but practical demands are always different from theory :)

I will mark this as blocked for now and wait for more requests.
